src/libsbn.cpp

```diff
@@ -85,6 +87,19 @@ void SBNInstance::TrainExpectationMaximization(double alpha, size_t em_loop_coun
       parent_to_range_, alpha, em_loop_count);
 }
 
+EigenVectorXd SBNInstance::CalculateSBNProbabilitiesInLog() {
+  // invariant: sbn_parameters_ are  in the log space i.e.,
+  // sbn_parameters_[s] = log P(S = s) = log(counts_s) - log(sum_{s'} counts_{s'})
```

Comment:
I'm not sure what this counts version is about... it's in terms of the SBN parameters, right?

Also, documentation about the use of functions/methods goes in header files. Comments in cpp files describe implementation details (as needed).

Reply:
The counts are just the number of times that topology is seen in the sample. Comments are now removed from the cpp files.